trackintel/geogr/distances.py

```diff
@@ -325,25 +282,15 @@ def calculate_haversine_length(gdf):
     return np.bincount((index[:-1])[no_mix], weights=dist[no_mix])
 
 
+@doc(
+    Positionfixes.get_speed,
+    first_arg="""
+Parameters
+----------
```

Comment:
Do we need the `Parameters` line here? All other docstrings do not have it.

Reply:
For to `pfs.get_speed()` we don't have any parameters so I excluded the section there.
But if we call it with ``ti.geogr.get_speed_positionfixes(pfs)` we now have a parameters and need the parameters section again. &rarr; thus in this special case we need the `Parameters`